packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,106 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
+  var homeIds = Homes.find().map(function (e) { return e._id; });
+  // remove current home from array
+  var indexOfCurrent = homeIds.indexOf(current_id);
+  if (indexOfCurrent > -1) {
+    homeIds.splice(indexOfCurrent, 1);
+  }
+  //return random element from rest of the homes
+  return homeIds[Math.floor(Math.random() * homeIds.length)];
+}
+  /*
+    Get random date from the range of today - startingpoint
+    @param start = months from today
+    creates two dates one of them being today
+    and second one being today minus months set in parameters
+  */
+function getRandomMoveInDate(start) {
+  var endDate = new Date();
```

Comment:
Use `const` for variable declarations where the reference will not change.

Reply:
Do you mean that the date variable should be constant?

Reply:
Basically, in ES2015+ all variables should be declared with `const` unless you intend to change what they reference. When the reference will change, use `let`. No more `var`.

See: https://ponyfoo.com/articles/var-let-const